scss/_db-ui.scss

```diff
@@ -0,0 +1,79 @@
+@import "variables";
+@import "font-faces";
+@import "typography-placeholder";
+@import "scaling-placeholder";
+
```

Comment:
Is this the bundled output for db ui base? Then we should add the colors as well?

Reply:
shouldn't we rename this to db_ui_base?

Reply:
fixed